src/redux/actions/index.js

```diff
@@ -0,0 +1,58 @@
+import store from "../store";
+export const ADD_ITEM = (product) => {
+  let newMenuState = [...store.getState().menuItems];
+  newMenuState = newMenuState.map((item) => {
+    if (item.Name === product.Name) {
```

Comment:
Prefer comparision with Id.
As name can create upparcase/lowerCase issue + there might be 2 items with almost same name